package.json

```diff
@@ -101,7 +101,7 @@
     "@storybook/react-webpack5": "^7.1.0-alpha.1",
     "@storybook/theming": "^7.1.0-alpha.1",
     "@testing-library/jest-dom": "^5.10.1",
-    "@testing-library/preact": "^3.2.3",
+    "@testing-library/preact": "^2.0.0",
```

Comment:
Console kept screaming that this package was the wrong version for preact. Downgrading it resolved terminal screams